### ISSUE TITLE
No Documentation for Setting Max Active Jobs

### ISSUE THREAD

**Issue by SilasReinagel** [zerocracy/farm#885]
https://github.com/zerocracy/farm/pull/693
https://github.com/zerocracy/datum/issues/258

Feature appears to be built, but there is no documentation on how to set User Options, such as Max Active jobs. 

There should be clear documentation on how to configure your Max Active Jobs.

0crat also does not list any commands that indicate how to do this. 

>   - apply Send your profile to the project, intending to join them
>   - breakup An opposite command to invite
>   - hello Just say hello
>   - invite Invite a friend
>   - quit Immediately exit the project
>   - rate Change your current hourly rate
>   - rfp Purchase a single RFP, 256 points will be charged
>   - vacation Change vacation mode
>   - version Print current version of the bot
>   - wallet Change your payment information

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
@silasreinagel[/z](https://www.0crat.com/u/silasreinagel) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by g4s8** [comment]
@SilasReinagel it is not implemented yet: https://github.com/zerocracy/farm/issues/820

**Comment by yegor256** [comment]
@SilasReinagel seems to be a duplicate of #820

**Comment by SilasReinagel** [comment]
What makes this a duplicate? It is blocked by #820, but completing #820 won't mean there is documentation for this particular feature.
